pkg/operator/configobserver/node/observe_latency_profile.go

```diff
@@ -67,6 +69,22 @@ func (l *latencyProfileObserver) observeLatencyProfile(
 		ret = configobserver.Pruned(ret, configPaths...)
 	}()
 
+	// if condition is not satisfied, return existing config back without any changes
+	// eg. shouldSuppressConfigUpdatesFunc is to check if rollouts are in progressing state
+	// then reject the update and do not set any newly observed values till the last rollout is complete
+	for _, shouldSupressConfigUpdatesFn := range l.shouldSuppressConfigUpdatesFuncs {
+		suppress, err := shouldSupressConfigUpdatesFn()
+		if err != nil {
+			errs = append(errs, err)
+			return existingConfig, errs
+		}
+		if suppress {
+			// log that latency profile couldn't be updated due to conditional
+			klog.Warningf("latency profile config observer did not update observed config due to unsatisified pre-requisite")
```

Comment:
Added the message about fromProfile -> toProfile rejection in status itself (`pkg), as it could spam the logs by printing the same log message over and over again.
https://github.com/openshift/library-go/blob/594ea24b635971bbd11e731bdc8f12ef9470010c/pkg/operator/latencyprofilecontroller/controller.go#L125-L137
https://github.com/openshift/library-go/blob/594ea24b635971bbd11e731bdc8f12ef9470010c/pkg/operator/latencyprofilecontroller/profilerejectionchecker.go#L83-L87

Reply:
Added `suppressReason` which is added in warning line.